src/StaticFAQ/FAQCategoryList.js

```diff
@@ -280,14 +284,16 @@ class RawFAQCategoryList extends React.Component<Props> {
   };
 
   render() {
-    const { categoryId, section, showGuaranteeArticle } = this.props;
+    const { categoryId, section, showGuaranteeArticle, language } = this.props;
+    const locale = fromLanguageToLocale(language);
 
     if (categoryId) {
       return (
         <QueryRenderer
           query={querySubcategory}
           render={this.renderSubcategory}
           variables={{ id: categoryId }}
+          createEnvironment={createEnvironment(null, locale)}
```

Comment:
our `QueryRenderer` doesn't accept such property, an environment is created inside of it automatically